zoo/services/assets/js/service_form.js

```diff
@@ -1,6 +1,14 @@
 import '../style/service_form.less'
+import { gitlabEnvs } from "./gitlab_envs.js"
 
 $(document).ready(function () {
+    //set default env type to zoo
+    $(".ui.segment.environment:visible").find('input[name$="type"]').each(function () {
```

Comment:
the only environments that should be possible to be added/edited via the ui are zoo ones, I'd rather remove the type input completely and set it directly on the backend

what do you think?

Reply:
I have removed it, yes i doesn't make sense to have it. Also i am thinking to remove type  from Environment model which I introduced because there is new model for gitlab environments called RepositoryEnvironment. And since both type of environments don't mix. I don't see the point to have property type on both models. What do you think?